test/fixtures/s3-fixture/src/main/java/fixture/s3/S3HttpHandler.java

```diff
@@ -243,7 +243,7 @@ public void handle(final HttpExchange exchange) throws IOException {
                 deletes.append("<DeleteResult>");
                 for (Iterator<Map.Entry<String, BytesReference>> iterator = blobs.entrySet().iterator(); iterator.hasNext(); ) {
                     Map.Entry<String, BytesReference> blob = iterator.next();
-                    String key = blob.getKey().replace("/" + path + "/", "");
+                    String key = blob.getKey().replace("/" + bucket + "/", "");
```

Comment:
This was a minor bug that we didn't notice before as most of the tests don't use a `base_path`